conference/src/main/java/com/kyriba/conference/domain/Presentation.java

```diff
@@ -0,0 +1,22 @@
+package com.kyriba.conference.domain;
+
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.time.LocalTime;
+
+
+//@Entity
+@Getter
+@ToString
+@EqualsAndHashCode
+public class Presentation
```

Comment:
It is impossible to populate such objects. :)

Reply:
it is not used in API - just to have a view how ScheduleService should look like